### PR TITLE
feat: landing page and category list page

### PR DESCRIPTION
**Pull request by vishal804**
Hello 

### Functionality 

Landing Page 
Category Listing Page 
Authentication

### Files to Review 
src/App.js
src/component/header/Header.js
src/context/auth-context.js
src/context/quiz-context.js
src/pages/authenticationPage/Signin.js
src/pages/authenticationPage/Signup.js
src/pages/categoryPage/CategoryPage.js
src/pages/homePage/HomePage.js
src/index.js
src/reducer/authReducer.js
src/reducer/quizReducer.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,19 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Header } from "./component";
+import { Routes, Route } from "react-router-dom";
+import { CategoryPage, HomePage, Signin, Signup } from "./pages";
```
Comment (Himanshu-Dhawale) [inline src/App.js:4]: I think you should create a separate route file and should import that file into App.js

**Review by Himanshu-Dhawale — APPROVED** [review]
1. UI looks good 
2. All classnames are self-explanatory
3. Functionalities are also working
4.  Just suggested a small change

**Review by hsnice16 — APPROVED** [review]
Good work. Couldn't find anything to suggest. Good to merge.

File: src/App.js (hunk 1 above)
Comment (vishal804) [reply src/App.js:4]: Will do that at time of some code refactor

**Pull request merged**